src/SCREENS.ts

```diff
@@ -46,6 +46,7 @@ const SCREENS = {
         ADVANCED_FILTERS_TAG_RHP: 'Search_Advanced_Filters_Tag_RHP',
         ADVANCED_FILTERS_FROM_RHP: 'Search_Advanced_Filters_From_RHP',
         ADVANCED_FILTERS_TO_RHP: 'Search_Advanced_Filters_To_RHP',
+        SAVED_SEARCH_RENAME_RHP: 'Search_Saved_Search_Rename_RHP',
```

Comment:
Let's add this to `CENTRAL_PANE_TO_RHP_MAPPING` too ([here](https://github.com/Expensify/App/blob/main/src/libs/Navigation/linkingConfig/CENTRAL_PANE_TO_RHP_MAPPING.ts#L52)), otherwise when reloading the central panel will switch to the chat pane.

Reply:
Good point, just added it. Going back to the bugs!